projects/rero/ng-core/src/lib/record/editor/type/password-generator-type.component.ts

```diff
@@ -100,6 +101,11 @@ interface PasswordGeneratorProps extends FormlyFieldProps {
     </p-inputGroupAddon>
     }
   </p-inputGroup>
+  @if (!showError && hasBeenGenerated) {
+    <div class="text-color-secondary my-3" translate>
```

Comment:
You can use a primeng message?

Reply:
It's just a gray line as an error message to inform that the line is copied to the clipboard.